reindexer/src/test/scala/uk/ac/wellcome/platform/reindexer/services/ReindexTrackerServiceTest.scala

```diff
@@ -23,18 +23,20 @@ class ReindexTrackerServiceTest
   it(
```

Comment:
I think we should add another test in this Suite that justifies the code change in RendexTrackerService

Reply:
👍 check that it actually references another shard ...